docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor driver plugin
+        File: setting-up-hypervisor-driver-plugin
```

Comment:
+1 to the name change from my side too, considering the addition of Hyper-V information.